### DISCUSSION TITLE
Cross-platform user interface

### DISCUSSION THREAD

**Discussion by Dolabok** [infinitered/ignite#2271] [General]
How do you manage the user interface of your application on wide screens?

For example, it would be weird to leave a mobile app tab bar on the web client instead of a navigation bar at the top of the screen. Instagram and wikipedia keep the burger menu on the left open all the time, which I personally find very ugly.

**Comment by frankcalise** [comment]
Hey @Dolabok - you can write platform specific code in a few ways

Check out [Expo's Platform Extensions](https://docs.expo.dev/workflow/glossary-of-terms/#platform-extensions) and also this [responsive blog post by Evan Bacon](https://blog.expo.dev/media-queries-with-react-native-for-ios-android-and-web-e0b73ed5777b)
